components/logo/Logo.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from "@storybook/react/types-6-0";
+import LogoMain from "./Logo_main";
+import LogoSmall from "./Logo_small";
+
+export default {
+  title: "Common/Logo",
+} as Meta;
+
+export const main = () => LogoMain();
+export const small = () => LogoSmall();
```

Comment:
Instead of creating two different components for your Logo, you can create a single component with a property to select either the main or small style.

e.g. (with JSX writing and `main`->`big`)

```tsx
export const big = () => <Logo size="big" />;
export const small = () => <Logo size="small" />;
```